run-tests.sh

```diff
@@ -1,73 +1,19 @@
 #!/usr/bin/env bash
 
-
 set -eu
 
-declare status_code
+remove_quotes () {
+  echo "$1" | tr -d '"'
+}
+
+declare error_code
 # shellcheck disable=SC2154
-# status_code="$(curl --silent --output /dev/null --write-out '%{http_code}' "$CFN_HelloWorldApi")"
-status_code="$(curl --silent --output /dev/null --write-out '%{http_code}' "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200")"
+#The CFN variables seem to include quotes when used in tests these must be removed before assigning them.
+CFN_CICBackendURL_NoQuotes=$(remove_quotes "$CFN_CICBackendURL")
+export DEV_CRI_CIC_API_URL=`echo ${CFN_CICBackendURL_NoQuotes%/}`
```

Comment:
will these github secrets: https://github.com/alphagov/di-ipv-cri-cic-api/blob/13e487a7e20da446839a748ea85de6de081d4ae9/.github/workflows/pull-request.yml#LL25C1-L27C52 still needed after this change ?

Reply:
Yes, this script is run inside the container and this is the only way I could get the values by using the outputs.